Makefile

```diff
@@ -239,6 +248,7 @@ bench: TESTTIMEOUT := $(BENCHTIMEOUT)
 
 .PHONY: check test testshort testrace testlogic bench
 check test testshort testrace bench: gotestdashi
+	$(warning You specified TESTS=$(TESTS) without specifying PKG - your tests will run against all packages)
```

Comment:
(This looks it's been superseded by L25-32.)

Reply:
Oops, yeah. Sloppy!